api/server/index.js

```diff
@@ -49,23 +52,34 @@ const startServer = async () => {
   app.use(noIndex);
   app.use(errorController);
   app.use(express.json({ limit: '3mb' }));
-  app.use(mongoSanitize());
   app.use(express.urlencoded({ extended: true, limit: '3mb' }));
-  app.use(staticCache(app.locals.paths.dist));
-  app.use(staticCache(app.locals.paths.fonts));
-  app.use(staticCache(app.locals.paths.assets));
-  app.set('trust proxy', trusted_proxy);
+  app.use(mongoSanitize());
   app.use(cors());
   app.use(cookieParser());
+  app.use(
+    helmet.contentSecurityPolicy({
+      directives: {
+        defaultSrc: ["'self'"], // allow everything from same origin by default
+        scriptSrc: ["'self'"], // only own scripts and a trusted external source
+        objectSrc: ["'none'"], // disallow plugin-based content
+        frameAncestors: ["'self'"], // prevent framing by other sites
+      },
+    }),
+  );
```

Comment:
fwiw, you can achieve this without adding a dependency. 

```suggestion
app.use((req, res, next) => {
  res.setHeader(
    'Content-Security-Policy',
    "default-src 'self'; script-src 'self'; object-src 'none'; frame-ancestors 'self'"
  );
  next();
});
```

Reply:
@mawburn I know that 😉, but I like the helmet package. but if @danny-avila does not want the package then I will off course remove it. but with the package there is more flexibility and a little easier for others to add new stuff to the header or the CSP setup in general.

Reply:
Cool, yeah he has the say obviously. I just noticed it as a change above my group's PR. 

I get the reasons for some things like this, but this one in particular feels very `left-pad` to me since the actual middleware is so simple and also more clear, imo.

Reply:
@danny-avila as discussed. keep using package [helmet](https://helmetjs.github.io/) for future easy integration or addition.

default security-related headers are disabled by using the `useDefaults: false`

Reply:
@danny-avila @rubentalstra 

I apologize if I wasn't clear with the library I mentioned. What I meant by: 

> feels very `left-pad` to me

Was that `left-pad` was a library that broke just about everything npm related back in 2016. The damages were in the billions of dollars literally overnight because some guy decided to take it down because nobody was supporting his open source.

https://en.wikipedia.org/wiki/Npm_left-pad_incident

It broke things like React, Babel, and a ton of other stuff because so many peer deps relied on it. The underlying issue of a developer being able to remove a package overnight like that was fixed almost immediately, so that's not relevant. 

What I meant by it was that the library itself was only like 11 lines of code and shouldn't have been used in the first place. It also highlighted the overuse of grabbing libraries to do every little thing. While people can't rip down repos from npm anymore, this is still sort of relevant if a new major version comes out or some kind of significant security hole or something is discovered. Plus the overall over reliance on 3rd party packages.

---

This might not be relevant to the decision to keep helmet, I'm cool with that if you think there are other benefits. I was just adding my 2¢ and wanted to clear up what I meant by what I said.